### ISSUE TITLE
slint-viewer does not show errors that happen after the initial project works

### ISSUE THREAD

**Issue by NigelBreslaw** [slint-ui/slint#11956]
### Bug Description

As requested by @ogoffart 

### Reproducible Code (if applicable)

```slint

```

### Environment Details

- Slint Version:
- Platform/OS:
- Programming Language:
- Backend/Renderer:
